### DISCUSSION TITLE
Laravel 10.15 + Livewire3 Beta - Cannot cache routes (fresh install)

### DISCUSSION THREAD

**Discussion by PrimDig** [livewire/livewire#5945] [7. Bugs (archived)]
Hello,

I believe there is an issue when attempting to cache laravel routes with the latest livewire package.

Steps to reproduce:

~ composer create-project laravel/laravel bugtest
~ cd bugtest
~ composer require livewire/livewire:^3.0@beta
~ php artisan optimize

Output:

   INFO  Caching the framework bootstrap files.  

  config ................................................................................................................................. 11ms DONE
  routes Segmentation fault <!- This is the error ->
  
  I can confirm this issue does not occur with a fresh install of Livewire2 with the latest version of Laravel.

**Comment by joshhanley** [comment]
@PrimDig thanks for reporting! This issue has already been reported and a failing test PR #5910 has been submitted, so we will continue the investigation there 🙂
